Source/Charts/Renderers/Scatter/CircleShapeRenderer.swift

```diff
@@ -60,4 +61,68 @@ open class CircleShapeRenderer : NSObject, ShapeRenderer
             context.fillEllipse(in: rect)
         }
     }
+    
+    func renderGradientShape(
```

Comment:
```suggestion
    func renderGradientShape(
```
```suggestion
    open func renderGradientShape(
```

Optional: Suggest we match the visibility of the method above.

Reply:
Ahh I actually left this as internal intentionally, it's not API I wanted to expose outside of the module